File: core/py/Order.py
class Order:
    def __init__(self,symbol,price,type,clientId,ticket,id):
        self.symbol=symbol
        self.price=price
        self.type=type
        self.openTime=0
        self.closeTime
        self.closePrice=0
        self.tp=0
        self.sl=0
        self.mode=0
        self.ticket=ticket
        self.clientId=clientId
        self.id=id

File: core/js/Order.js
class Order{
  constructor(symbol,price,type,clientId,ticket,id){
    this.symbol=symbol; this.price=price; this.type=type;
    this.openTime=0; this.closeTime; this.closePrice=0;
    this.tp=0; this.sl=0; this.mode=0;
    this.ticket=ticket; this.clientId=clientId; this.id=id;
  }
}
